### PR TITLE
`selector-pseudo-class-case` rule

### PR DESCRIPTION
**Pull request by evilebottnawi**

### CODE REVIEW

File: src/rules/selector-pseudo-class-case/index.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      possible: [
+        "lower",
+        "upper",
+      ],
+    })
+    if (!validOptions) { return }
+
+    root.walkRules(rule => {
+      const selector = rule.selector
+      const startIndexPseudoElement = selector.indexOf(":")
```
Comment (jeddy3) [inline src/rules/selector-pseudo-class-case/index.js]: Did you mean "PseudoClass" here and else where, rather than "PseudoElement"?

**Comment by jeddy3** [comment]
Besides a question about a variable name, this looks great to me!

Can you rebase please.

**Comment by evilebottnawi** [comment]
@jeddy3 done

**Comment by jeddy3** [comment]
Thanks for the rebase. 

Do you want to rename the `pseudoElement` variable to `pseudo` as am I right in thinking that it can be either a pseudo element or pseudo class in this context?

**Comment by evilebottnawi** [comment]
@jeddy3 good?

**Comment by jeddy3** [comment]
Much clearer thanks!

**Pull request merged**